### DISCUSSION TITLE
console_test now showing data on graph

### DISCUSSION THREAD

**Discussion by rothdebian** [espressif/esp-csi#217] [Q&A]
I followed the instructions on the Readme.md from console_test (https://github.com/espressif/esp-csi/blob/master/examples/esp-radar/console_test/README.md).

The first difficult is that on step 3.2 it asks to flash on device console_test but as showed on 2.1 it should be csi_recv or csi_recv_router. I tried both for test but when I run console_test/tools/esp_csi_toool.py on esp32 with csi_recv or csi_recv_router I see that files are created under log directory.  The program created four files: csi_data.csv, device_info.csv, log_data.txt and radar_data.csv but the only file that receive data is log_data.txt.

Using csi_recv_router I can see the device connected on router admin interface and some data on log_data.txt but no graph is filled with information.

I tried to compile csi_recv_router on windows 11 and debian linux 12 and it does not change. I tried to run esp_csi_tool.py on both operation systems and can not get any data on graph.

Is there anything that I can do to try to understand what's wrong ?

thanks a lot

**Comment by RikeshMMM** [comment]
Facing the same issue, were you able to figure out the solution?

**Reply by rothdebian** [reply]
> Facing the same issue, were you able to figure out the solution?

I'm still investigating what happened. I have a clue but nothing that I can say to help at this moment.

**Comment by NingYuxiang** [comment]
When using the console_test project, Device 1 should burn the csi_send project, and Device 2 should burn the console_test project. Figure 2.1 only indicates the transmitter and receiver, not the project names.
The console_test/tools/esp_csi_tool.py is intended to be used with the console_test project, and using it with other projects may result in data corruption or errors. Please make sure to match the correct usage.

**Reply by rothdebian** [reply]
> When using the console_test project, Device 1 should burn the csi_send project, and Device 2 should burn the console_test project. Figure 2.1 only indicates the transmitter and receiver, not the project names. The console_test/tools/esp_csi_tool.py is intended to be used with the console_test project, and using it with other projects may result in data corruption or errors. Please make sure to match the correct usage.

Thank you for your help, but I am sure that I'm using it right. 
I'm trying to understand why everything works fine when I got and onl menuconfig. With and old configuration from a friend everything runs fine. I'm looking on changes to find wich configuration is messing all the code.

**Reply by NingYuxiang** [reply]
Could it be an issue with the baud rate? The esp_csi_tool.py requires the ESP chip to be configured to use a baud rate of 921600.

**Reply by rothdebian** [reply]
> Could it be an issue with the baud rate? The esp_csi_tool.py requires the ESP chip to be configured to use a baud rate of 921600.

Thank your for your atention but the configuration that works on an very old version is use the same baud rate. I'm still looking for the point that generate this situation.
